app/models/dialog_field_importer.rb

```diff
@@ -32,12 +32,15 @@ def import_field(dialog_field_attributes)
 
   def set_category_for_tag_control(dialog_field, dialog_field_attributes)
     category_name = dialog_field_attributes["options"][:category_name]
-    if category_name
+    category_id = dialog_field_attributes["options"][:category_id]
+    if category_id
+      dialog_field.category = category_id
```

Comment:
Dunno if this needs the description as well.

Reply:
@eclarizio - Does it need the description?